### PR TITLE
Format strings

### PR DESCRIPTION
**Pull request by Cabalist**
Converts % formatting to `.format()`.  This has an effect on the translations.  I have updated all the *.po files but the *.mo files will need to be regenerated.  I am not as familiar with gettext as I probably should be to do this commit.  I believe that the the workflow is that individual language files (.po) are create off the .pot file and then customized.  The .mo are a generated binary file that the application looks up based on the systems LANG or LC_MESSAGES.  Please correct me if I am wrong!

### CODE REVIEW

File: messages/gl/LC_MESSAGES/comix.po [hunk 1]
```diff
@@ -3,7 +3,7 @@ msgstr ""
 "Project-Id-Version: \n"
 "POT-Creation-Date: \n"
 "PO-Revision-Date: \n"
-"Last-Translator: Marcelo G�es <vanquirius@gentoo.org>\n"
+"Last-Translator: Marcelo G�es <vanquirius@gentoo.org>\n"
```
Comment (DavidPineauScality) [inline messages/gl/LC_MESSAGES/comix.po]: what is this change ? Did you change the file's encoding somehow ?
Github/my browser are not rendering it properly anymore ...

File: messages/pt_BR/LC_MESSAGES/comix.po [hunk 2]
```diff
@@ -3,7 +3,7 @@ msgstr ""
 "Project-Id-Version: \n"
 "POT-Creation-Date: \n"
 "PO-Revision-Date: \n"
-"Last-Translator: Marcelo G�es <vanquirius@gentoo.org>\n"
+"Last-Translator: Marcelo G�es <vanquirius@gentoo.org>\n"
```
Comment (DavidPineauScality) [inline messages/pt_BR/LC_MESSAGES/comix.po]: Same here, encoding seems to have been changed. I don't think this is good :/

File: messages/sv/LC_MESSAGES/comix.po [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # Svensk katalog.
-# �versatt 2005-2006 av
+# �versatt 2005-2006 av
```
Comment (DavidPineauScality) [inline messages/sv/LC_MESSAGES/comix.po]: Here again, encoding seems to have changed

**Review by ghost — CHANGES_REQUESTED** [review]
pretty impressive changes again !
That being said, only worry is that there seem to have been 3 files (if I didn't miss any) that changed encoding (like I'm not displaying them properly now in my browser though they were before the change).

Any idea why ? Think you can fix this ?

**Comment by Cabalist** [comment]
I see what happened.  The files were in a non-UTF-8 format and my editor just blasted them away.  Working on a fix now.

**Review by Joacchim — APPROVED** [review]
Yup, looks much better, thanks; even pitching in some additional format fixes  !

**Pull request merged**